File: Modul 1/Soal 10/PRAK110-NIM-Nama.c
#include <stdio.h>
#include <math.h>

int main(){
    int A = 12, B, C = 5, Keliling, Luas;
    B = (int) sqrt(pow(A, 2) + pow(C, 2));
    Keliling = A + B + C;
    Luas = (A * C) / 2;

    printf("Diketahui : \n");
    printf("Alas = %d cm\n", C);
    printf("Tinggi = %d cm\n", A);
    
    printf("\n");

    printf("Jawab : \n");
    printf("Sisi A = %d cm\n", A);
    printf("Sisi B = %d cm\n", B);
    printf("Sisi C = %d cm\n", C);
    printf("Keliling = %d cm\n", Keliling);
    printf("Luas = %d cm\n", Luas);
}

File: Modul 1/Soal 10/PRAK110-NIM-Nama.py
import math

A = 12
C = 5
B = math.sqrt(A**2 + C**2)
Keliling = A + B + C
Luas = (A * C) / 2

print("Diketahui :")
print(f"Alas = {C} cm")
print(f"Tinggi = {A} cm")

print()

print("Jawab :")
print(f"Sisi A = {A} cm")
print(f"Sisi B = {B:.0f} cm")
print(f"Sisi C = {C} cm")
print(f"Keliling = {Keliling:.0f} cm")
print(f"Luas = {Luas:.0f} cm")